aws/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+provider "aws" {
+    access_key = "${var.access_key}"
```

Comment:
What's better var or Environment Variables?

Reply:
Trying to keep my secrets out of main.tf with variables. Appears folks keep secrets in terraform.tfvars so that’s what I did. I don’t know how this is supposed to be setup, just going from AWS terraform examples and trying to figure it out.